components/member-profile/member-profile.module.scss

```diff
@@ -107,12 +107,37 @@
   font-size: 16px;
 }
 
+.getIntroButton {
+  font-size: 1.2rem;
+  padding: 8px 30px;
+  background-color: rgb(238, 26, 117);
+  color: #fff;
+  border-radius: 8px;
+  transition-duration: 0.4s;
+  border: rgb(238, 26, 117);
+}
+
+.getIntroButton:hover {
+  background-color: #fff;
+  color: rgb(238, 26, 117);
+  border-radius: 8px;
+  border: 1px solid rgb(238, 26, 117);
+}
+
+.modalCloseOff {
```

Comment:
cannot find this class name in the `member-profile/index.js` file. Can we please remove it?

Also, the name looks pretty vague.